src/graphql/data/nft/Asset.ts

```diff
@@ -128,7 +128,7 @@ function formatAssetQueryData(queryAsset: NftAssetsQueryAsset, totalCount?: numb
   const asset = queryAsset.node
   const ethPrice = parseEther(wrapScientificNotation(asset.listings?.edges[0]?.node.price.value ?? 0)).toString()
   return {
-    id: asset.id,
+    id: undefined,
```

Comment:
the id being passed here is the one assigned by the backend but we need to assign one ourselves when we add it to bag

Reply:
honestly I don't really know what the purpose of that would be if we assign a new one when it gets added to the bag. Previously we only gave an asset an id when it was added to the bag